checks.d/docker_daemon.py

```diff
@@ -672,6 +673,12 @@ def _format_events(self, aggregated_events, containers_by_id):
                 status_changes="\n".join(
                     ["%s \t%s" % (change[1].upper(), change[0]) for change in status_change])
             )
+
+            if any(e in status_text for e in error_alert_type):
```

Comment:
nitpicking because in this case it's not a real problem, but in general be careful with single-char variable names as some characters are used by [pdb](https://docs.python.org/2/library/pdb.html#debugger-commands), and it makes debugging difficult.